pkgs/development/libraries/libvirt/default.nix

```diff
@@ -32,19 +32,19 @@ let
   };
 in stdenv.mkDerivation rec {
   pname = "libvirt";
-  version = "7.0.0";
+  version = "7.5.0";
 
   src =
     if buildFromTarball then
       fetchurl {
         url = "https://libvirt.org/sources/${pname}-${version}.tar.xz";
-        sha256 = "12fxkpy7j2qhfxypw9jg3bzdd9xx6vf6x96iy5kjihh89n236f6a";
+        sha256 = "sha256-uKEu9zSg6HrHx7yb8H25Yvt8r6yAGcXVTGNCZc+CDNY=";
       }
     else
       fetchgit {
```

Comment:
Can we use fetchFromGitLab here?

Reply:
Wanted to, but it seems like fetchFromGitLab does not have fetchSubmodules...

Reply:
I added those options in #131047